course5/homework2/newton/newton.cc

```diff
@@ -0,0 +1,58 @@
+#include <iostream>
+#include <string>
+
+// Newton method of sqrt. The stack frame was 48 for Debug mode and 64 for Release mode.
+
+bool* start = nullptr;
+bool* end = nullptr;
+
+float function( float y, int it, float& rx, int& ri)
```

Comment:
"function" is not the best name for a function calculating square root. Make your API such so you'd enjoy using it.